### ISSUE TITLE
文档”Categorical.probs()方法”的错误修改建议

### ISSUE THREAD

**Issue by yuxiaooye** [PaddlePaddle/Paddle#45560]
### 文档链接&描述 Document Links & Description

https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/distribution/Categorical_cn.html

对于Categorical.probs()方法：
- `logtis`拼错
- `value[:-1] = logits[:-1]`存在问题，例如当logits是2D tensor时：

```python
logits = paddle.to_tensor([
    [0.8, 0.2],
    [0.4, 0.6]]
    )
value = paddle.to_tensor([
    [1],
    [0]]
    )
```
logits[:-1]为[0.8, 0.2]，value[:-1]为[1]，并不相等

### 请提出你的建议 Please give your suggestion

应将`value[:-1] = logits[:-1]`修改为`value.shape[:-1] = logits.shape[:-1]`

**Comment by luotao1** [comment]
@Insomnia-y 
- 欢迎提一个PR来修复下问题
- 飞桨正在举行迎新年快乐开源活动，有兴趣可以参加下：https://aistudio.baidu.com/aistudio/activitydetail/1503019026

**Comment by Ligoml** [comment]
开发者你好，非常欢迎你直接提pr来解决这个问题哈，你可以在pr中@我，我会及时处理~

这里是一些参考链接：

[飞桨官网-贡献指南-代码贡献流程](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/dev_guides/code_contributing_path_cn.html)
[飞桨官网-贡献指南-新增API开发&提交流程-API文档书写规范](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/dev_guides/api_contributing_guides/api_docs_guidelines_cn.html)
参考PR：https://github.com/PaddlePaddle/Paddle/pull/42942 ； https://github.com/PaddlePaddle/docs/pull/4850

**Comment by enkilee** [comment]
fix it :
https://github.com/PaddlePaddle/docs/pull/5558
